plugins/rebalance/transaction.go

```diff
@@ -0,0 +1,48 @@
+package rebalance
+
+import (
+	"github.com/gluster/glusterd2/glusterd2/daemon"
+	"github.com/gluster/glusterd2/glusterd2/transaction"
+	rebalanceapi "github.com/gluster/glusterd2/plugins/rebalance/api"
+)
+
+type actionType uint64
+
+const (
+	actionStart actionType = iota
+	actionStop
+)
+
+func rebalanceCommand(c transaction.TxnCtx, cmd actionType) error {
+	var rinfo rebalanceapi.RebalanceInfo
+	err := c.Get("rinfo", &rinfo)
+	if err != nil {
+		return err
+	}
+
+	rebalanceProcess, err := NewRebalanceProcess(rinfo)
+	if err != nil {
+		return err
+	}
+
+	switch cmd {
+
+	case actionStart:
+		var r rebalanceapi.RebalanceInfo
+		// Reset all variables
+		glusterdvolinforesetstats(r)
```

Comment:
By default all values will be zero. Do we need this function. Also if you are modifying rebalanceinfo you should send pointer. `glusterdvolinforesetstats(&r)`